### PR TITLE
Relaxed scikit-image requirement, edited readme

### PR DESCRIPTION
**Pull request by JennyVoPhamhi**
Made these changes for the reasons described in the email I sent just now!

### CODE REVIEW

**Review by kevinyamauchi — COMMENTED** [review]
I think this looks good. You can specify the python versions in the setup.py file. I think it would be good to do so. See here: https://packaging.python.org/guides/distributing-packages-using-setuptools/#python-requires

**Comment by JennyVoPhamhi** [comment]
Hi Kevin,

Thank you for this feedback and guidance! I have updated the setup.py file accordingly. Do you think we can go ahead and merge to master?

Thank you!
Jenny

More details:

Extrapolating from the examples here (https://packaging.python.org/guides/distributing-packages-using-setuptools/#python-requires) I added the line
`python_requires=‘==3.6.*, ==3.7.*`

And got 

ERROR: Package 'instapipeline' requires a different Python: 3.7.10 not in '==3.6.*,==3.7.*'

After trying various things, what ended up working was `python_requires='>=3.6, <3.8’,`.

I tried running `pip3 install .` in the repo in different python environments just to check for expected behavior:
- In a python 3.5 environment it gives a pandas installation error
- In a python 3.6 environment it worked, and importing instapipeline, starfish, etc. in the python3 interpreter worked
- In a python 3.7 environment it worked, and importing instapipeline, starfish, etc. in the python3 interpreter worked
- In a python 3.8 environment it gave a python version error: ERROR: Package 'instapipeline' requires a different Python: 3.8.8 not in '<3.8,>=3.6'

File: setup.py [hunk 1]
```diff
@@ -9,6 +9,8 @@
 
 setup(
 	name='instapipeline',
+	# python_requires='>=3.6, !=3.8.*, !=3.9.*',
```
Comment (kevinyamauchi) [inline setup.py]: Can you please remove this commented out line?

**Review by kevinyamauchi — APPROVED** [review]
Thanks, @JennyVoPhamhi ! Looks great to me. Please just seem my very minor comment below. Please feel free to merge after addressing that comment.

File: setup.py (hunk 1 above)
Comment (JennyVoPhamhi) [reply setup.py]: Done! Thank you for catching this!

**Pull request merged**